foreman/foreman.spec

```diff
@@ -223,6 +223,12 @@ BuildRequires: libuv
 BuildRequires: nodejs-packaging
 
 # package.json devDependencies
+#BuildRequires: npm(@storybook/addon-actions) >= 3.2.12
+#BuildRequires: npm(@storybook/addon-actions) < 4.0.0
+#BuildRequires: npm(@storybook/react) >= 3.2.12
```

Comment:
huh?

Reply:
I'm not sure what to do about these - they've been here for a while but we never packaged storybooks until now.